libsolidity/analysis/OverrideChecker.cpp

```diff
@@ -140,7 +140,8 @@ vector<ContractDefinition const*> resolveDirectBaseContracts(ContractDefinition
 		Declaration const* baseDecl =
 			specifier->name().annotation().referencedDeclaration;
 		auto contract = dynamic_cast<ContractDefinition const*>(baseDecl);
-		solAssert(contract, "contract is null");
+		if (!contract)
```

Comment:
I'm not sure I would like to keep this without the "recover errors" setting switched on.

Reply:
Just removed the assertion. Hope this will not generate a problem...